lib/oneview-sdk/resource/api200/logical_switch.rb

```diff
@@ -47,11 +47,10 @@ def create
       end
 
       # Reclaims the top-of-rack switches in a logical switch.
-      # @return [OneviewSDK::LogicalSwitch] self
-      def refresh_state!
+      # @return [Hash] Http response body with logical switch data.
```

Comment:
The REST call returns only a task, not the logical switch. I guess you have two alternatives:
 
1. Remove the comment with the `@return` since it would be useless.
2. After the REST call use the `refresh` or `retrieve!` method to update the resource